forge-gui/res/cardsfolder/upcoming/dunbarrow_revivalist.txt

```diff
@@ -0,0 +1,15 @@
+Name:Dunbarrow Revivalist
+ManaCost:1 B
+Types:Creature Human Warlock
+PT:3/1
+K:Bargain
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigBoon | TriggerDescription$ When CARDNAME enters the battlefield, you get a one-time boon with "When one or more creatures enter the battlefield under your control, create a Wicked Role token attached to one of them." 
+SVar:TrigBoon:DB$ Effect | Boon$ True | Duration$ Permanent | Triggers$ ChangeZoneTrig | SubAbility$ DBBargained | SpellDescription$ You get a one-time boon with "When one or more creatures enter the battlefield under your control, create a Wicked Role token attached to one of them."
+SVar:ChangeZoneTrig:Mode$ ChangesZoneAll | ValidCards$ Creature.YouCtrl | Destination$ Battlefield | TriggerZones$ Battlefield | ActivationLimit$ 1 | Execute$ TrigChoose | TriggerDescription$ When one or more creatures enter the battlefield under your control, create a Wicked Role token attached to one of them
```

Comment:
it shouldn't even work if you give it the wrong TriggerZone (though effect API might work around it in the background)